src/ceph-disk

```diff
@@ -2191,11 +2353,65 @@ def mount_activate(
 
     path = mount(dev=dev, fstype=fstype, options=mount_options)
 
+    # check if the disk is deactive, change the journal owner, group
+    # mode for correct user and group.
+    if os.path.exists(os.path.join(path, 'deactive')):
+        # logging to syslog will help us easy to know udev triggered failure
+        if not reactivate:
+            unmount(path)
```

Comment:
If osd start failure on machine boot stage, we can not see some failure reason.
due to above reason, I prefer use syslog.syslog to show the failure cause by deactivated.

Or we could see the log message when we use LOG.info on machine boot stage?